types/lru-cache/index.d.ts

```diff
@@ -1,12 +1,12 @@
-// Type definitions for lru-cache 7.4
+// Type definitions for lru-cache 7.5
 // Project: https://github.com/isaacs/node-lru-cache
 // Definitions by: Bart van der Schoor <https://github.com/Bartvds>
 //                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
 declare class LRUCache<K, V> implements Iterable<[K, V]> {
-    constructor(options?: LRUCache.Options<K, V>);
+    constructor(options: LRUCache.Options<K, V>);
```

Comment:
is `options` no longer optional starting at 7.5 or was it never optional and DT was wrong? I can't seem to find a change related to that on the original library (https://github.com/isaacs/node-lru-cache/compare/v7.4.0...v7.5.0)

Reply:
It was never optional and the typings are wrong :(

See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59162